### DISCUSSION TITLE
"Removed Button control for DataObjects layout definition" in Pimcore 11

### DISCUSSION THREAD

**Discussion by jantomicky** [pimcore/pimcore#16587] [Q&A / Support]
Hello, as per title, what is the recommended course of action here? We use buttons in our class definitions, what do we do now, after the upgrade, use [the Event Listener UI](https://pimcore.com/docs/platform/Pimcore/Extending_Pimcore/Bundle_Developers_Guide/Event_Listener_UI/)? Thanks!

**Comment by jantomicky** [comment]
Please, anyone?

**Comment by fashxp** [comment]
see also here: https://github.com/pimcore/pimcore/issues/15124

**Comment by jantomicky** [comment]
Thank you for the response. After considering this, I think the simplest way will be adding a button via the Event Listener UI, so we will do that.
